<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashBoard</class>
 <widget class="QMainWindow" name="DashBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生成绩管理系统 | 控制台</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/ico/images/appstu.ICO</normaloff>:/ico/images/appstu.ICO</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-image: url(:/jpg/images/main.jpg);</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>基础设置</string>
    </property>
    <addaction name="majorSettingAction"/>
    <addaction name="classSettingAction"/>
    <addaction name="subjectSettingAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>基本信息管理</string>
    </property>
    <addaction name="studentDataAction"/>
    <addaction name="scoreDataAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>系统查询</string>
    </property>
    <addaction name="studentSearchAction"/>
    <addaction name="scoreSearchAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>系统管理</string>
    </property>
    <addaction name="adminSettingAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>数据分析</string>
    </property>
    <addaction name="scoreAnalysisAction"/>
    <addaction name="majorAnalysisAction"/>
   </widget>
   <widget class="QMenu" name="menuAI">
    <property name="title">
     <string>AI大模型</string>
    </property>
    <addaction name="aiChatAction"/>
    <addaction name="aiScoreAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menuAI"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="majorSettingAction">
   <property name="text">
    <string>专业设置</string>
   </property>
  </action>
  <action name="classSettingAction">
   <property name="text">
    <string>班级设置</string>
   </property>
  </action>
  <action name="subjectSettingAction">
   <property name="text">
    <string>课程设置</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>考试类别设置</string>
   </property>
  </action>
  <action name="studentDataAction">
   <property name="text">
    <string>学生信息管理</string>
   </property>
  </action>
  <action name="scoreDataAction">
   <property name="text">
    <string>成绩管理</string>
   </property>
  </action>
  <action name="studentSearchAction">
   <property name="text">
    <string>学生信息查询</string>
   </property>
  </action>
  <action name="scoreSearchAction">
   <property name="text">
    <string>学生成绩查询</string>
   </property>
  </action>
  <action name="adminSettingAction">
   <property name="text">
    <string>用户信息维护</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="scoreAnalysisAction">
   <property name="text">
    <string>学生成绩分析</string>
   </property>
  </action>
  <action name="majorAnalysisAction">
   <property name="text">
    <string>专业人数分析</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>专业查询</string>
   </property>
  </action>
  <action name="aiChatAction">
   <property name="text">
    <string>智能机器人</string>
   </property>
  </action>
  <action name="aiScoreAction">
   <property name="text">
    <string>AI模型分析成绩</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>DashBoard</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
 </slots>
</ui>
